### ISSUE TITLE
Moving to Meteor Community

### ISSUE THREAD

**Issue by CaptainN** [Meteor-Community-Packages/ground-db#212]
HI @raix! I'd like to help maintain the Ground DB project, since I use it in just about every project I work on (and have incorporated it into my [`npdev:collections`](https://github.com/CaptainN/npdev-collections) package). I don't actually think it needs much work to keep it updated - mostly just needs someone to merge PR #211 and do a release.

To that end, would you be open to migrating the project to the [Meteor Community Packages](https://github.com/Meteor-Community-Packages/organization) group? The idea is that once there, we can get a set of reliable professionals to help with that sort of maintenance. For this project (and perhaps the ones on which is relies) I'd be happy to take point.

I look forward to hearing from you.  By the way, I'm happy to chat here, or via [email](Kevin@unFocus.com) or Slack (we have a [Meteor Community](meteor-community.slack.com) Slack channel), or any other way about any of this.

**Comment by raix** [comment]
Hi @CaptainN I'm glad to hear - It would be great to get some maintainers on it, I'm open for moving it to the community group. I don't use Meteor anymore since I'm on a completely different stack (I do miss it from time to time...) :)

**Comment by raix** [comment]
The build failes due to outdated meteor token (the release is otherwise configured to do semantic releases)

**Comment by CaptainN** [comment]
Sorry for the long horizon on these communications. Busy busy. I'm thinking it'll be easier to manage in the Community Packages group, since we have a number of maintainers over there. Would you mind if I move it there?

Also, what do you think about moving the projects on which Ground:DB depends as well - specifically, `raix:eventstate` and `raix:eventemitter`?

**Comment by CaptainN** [comment]
Any more thoughts to this? I'd like to get at least the few updates I have in there already released, since they are not blocking updating core components (like mongo and minimongo) to the latest versions in projects that use this. I'd be happy to work in this group directly. Whatever you like.

**Comment by CaptainN** [comment]
@raix  I'd like to release 2.0.1 (or maybe 2.1.0) with the removal of underscore, and raix:eventstate and local-forage version bumps, but I'm not sure how to do that. Do I simply merge that to master to trigger a release?

Is there something I can do to fix the outdated token you mentioned above? Or is that something you need to do in the Meteor account system? My Meteor user is CaptainN if inviting me to that will help get me access to fix that.

**Comment by raix** [comment]
Maybe we should move those repos to the Meteor Community too? (maybe remove the "raix" prefix not sure how deprecation of packages work)

**Comment by CaptainN** [comment]
We've been trying to keep the same names if we can, so that existing users can just upgrade everything.

Part of that is getting us access to be able to publish through atmosphere. I'd like to do a release for ground:db, but I don't think I have access to renew the token, or whatever else I'll need.

**Comment by CaptainN** [comment]
@raix I think all I need to be able to make a release is access to the `ground` org on [meteor.com](https://www.meteor.com/account-settings). Any chance you can help me out with that? My meteor username is `captainn` (2 "n"s).

**Comment by raix** [comment]
@CaptainN I'm really sorry for not returning sooner with any updates - been a crazy year
I'm cutting down on open source work, simply don't have the time/energy as when I was younger. I'll be closing the meteor account, as I haven't been active for a couple of years now.

I'd like to remove the GroundMeteor organsation
It seems like db package depends on `raix:eventstate` not sure if I should move that package over? (not sure if other packages are needed)

**Comment by raix** [comment]
@CaptainN when trying to transfer - what teams should I select?
(I'll also transfer raix:eventstate and raix:eventemitter)

**Comment by CaptainN** [comment]
Hi! Thanks for the effort, I appreciate it.

I was hoping to continue to make releases with the same namespace, but if that can't happen, I can probably just fork all these over to another org. I don't know what the best way forward is.

The GitHub community [is here](https://github.com/Meteor-Community-Packages). I'm not sure which group in Atmosphere, but I'll find out.

**Comment by copleykj** [comment]
@raix If you want to just go ahead and transfer all of them to @Meteor-Community-Packages org, one of us will take care of setting up teams and what not. The only other thing we'll require you to do is add the communitypackages Meteor org as a maintainer `meteor admin maintainers <package name> --add comunitypackages`

**Comment by brunoneumann** [comment]
Hello guys! Is there any future here? I recently upgraded my app to Meteor 2.5 and GroundDB stopped working with no errors apparently, just not connects with offline database anymore.

**Comment by StorytellerCZ** [comment]
@raix I'm ready to move this. Let me know when you have time to make this happen.

**Comment by banjerluke** [comment]
I rewrote ground:db in Typescript using Dexie instead of localforage (massive speed increase) with fixes for Meteor 2.4+, keeping pretty much the same API. I posted the code on [this forum thread](https://forums.meteor.com/t/is-grounddb-the-best-for-offline-storage/56654/16). It's been in production for a few months now with no issues. It's just a couple classes right now since I didn't quite know how to package it up, but if there's interest in adding it to the Meteor community packages repo, let me know (or just take the code and run with it).

**Comment by StorytellerCZ** [comment]
@banjerluke yes, I just did update ground:db for Meteor 3, but adding Typescript and other improvements are well worth considering.
@raix I just tried to publish new version and encountered an issue that `communitypackages` org doesn't have access rights, we had this issue elsewhere as well. We need to make sure that: 
```
meteor admin maintainers <package name> --add communitypackages
```
has been applied on all the troublesome packages.
